OAK-Marketplace-Emotion-Recognition/utils/inference.py

```diff
@@ -0,0 +1,25 @@
+from emotion_recognition_retail import process_frame
+import cv2
+import numpy as np
+
+
+def process_cam(model, emotion_analyzer, show: bool = True):
+    visualization_results = []
+    model.add_cam_to_pipeline()
+    proceed = True
+    while proceed:
+        input_width, input_height = model.get_input_shapes()
+        image = np.ascontiguousarray(
+            model.get_frame_from_camera()
+                .reshape((3, input_height, input_width))
+                .transpose(1, 2, 0),
+        )
+        ret, proceed, _ = process_frame(image, model, visualization_func=None)
```

Comment:
Original process_frame function returns 2 values, not 3, so this code won't be working. 
```python
def process_frame(image, model, visualization_func):
    image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
    ret = model.process_sample(Image.fromarray(image))
    proceed = True
    if visualization_func is not None:
        vis_result = visualization_func(image, ret)
        cv2.imshow("Visualization", vis_result[:, :, ::-1])
        if cv2.waitKey(1) == ord("q"):
            cv2.destroyAllWindows()
            proceed = False
    return ret, proceed
```